### PR TITLE
Fixed Istio-security linter errors

### LINKED ISSUE #1446: Istio-security linter errors
```
I1103 22:41:44.803] security/cmd/node_agent/na/nodeagent_test.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:44.803] security/pkg/pki/ca/ca.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:44.804] security/pkg/pki/ca/ca_test.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:44.804] security/pkg/pki/ca/controller/secret.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:44.804] security/pkg/pki/ca/controller/secret_test.go:1::warning: file is not gofmted with -s (gofmt)
I1103 22:41:53.633] security/cmd/node_agent/na/nodeagent_test.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.634] security/pkg/pki/ca/ca.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.672] security/pkg/pki/ca/ca_test.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.672] security/pkg/pki/ca/controller/secret.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.673] security/pkg/pki/ca/controller/secret_test.go:1::warning: file is not goimported (goimports)
I1103 22:41:53.791] security/cmd/node_agent/na/nafactory.go:35:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.791] security/cmd/node_agent/na/nodeagent.go:43:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.792] security/cmd/node_agent/na/nodeagent.go:51:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.792] security/cmd/node_agent/na/nodeagent_test.go:50:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.792] security/cmd/node_agent/na/nodeagent_test.go:145:50:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.792] security/pkg/pki/crypto.go:34:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.792] security/pkg/pki/crypto.go:39:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.793] security/pkg/pki/crypto.go:50:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.793] security/pkg/pki/crypto.go:54:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.793] security/pkg/pki/crypto.go:63:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.793] security/pkg/pki/crypto.go:70:27:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.793] security/pkg/pki/crypto.go:76:27:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.794] security/pkg/pki/crypto.go:80:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.794] security/pkg/pki/san.go:92:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.794] security/pkg/pki/san.go:104:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.794] security/pkg/pki/san.go:111:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.794] security/pkg/pki/san.go:116:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.795] security/pkg/pki/testutil/util.go:101:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.795] security/pkg/pki/testutil/util.go:105:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.795] security/pkg/pki/testutil/util.go:109:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.795] security/pkg/pki/testutil/util.go:113:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.795] security/pkg/pki/testutil/util.go:117:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.795] security/pkg/pki/testutil/util.go:121:21:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.796] security/pkg/platform/aws.go:90:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.796] security/pkg/platform/aws.go:95:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.796] security/pkg/platform/aws.go:100:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.796] security/pkg/platform/aws.go:105:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.796] security/pkg/platform/aws.go:110:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.797] security/pkg/platform/aws.go:121:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.797] security/pkg/platform/aws.go:126:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.797] security/pkg/platform/client.go:57:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.797] security/pkg/platform/onprem.go:68:25:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.798] security/pkg/platform/onprem.go:77:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.798] security/pkg/platform/onprem.go:95:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.798] security/pkg/platform/onprem.go:102:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
I1103 22:41:53.798] security/pkg/platform/onprem.go:107:26:warning: error strings should not be capitalized or end with punctuation or a newline (golint)
```

### PR DESCRIPTION
**Pull request by JimmyCYJ**
**What this PR does / why we need it**: Fix linter errors.

**Which issue this PR fixes** : fixes #1446 

**Special notes for your reviewer**:

**Release note**:
Fixed Istio-security linter errors.

```release-note
Fixed linter errors.
```

### CODE REVIEW

**Comment by wattli** [comment]
Thanks for doing this, please fix the presubmit.sh https://k8s-gubernator.appspot.com/build/istio-prow/pull/istio_istio/1787/istio-presubmit/386/

File: security/cmd/node_agent/na/nodeagent.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ type cAGrpcClientImpl struct {
 // SendCSR sends CSR to CA through GRPC.
 func (c *cAGrpcClientImpl) SendCSR(req *pb.Request, pc platform.Client, cfg *Config) (*pb.Response, error) {
 	if cfg.IstioCAAddress == "" {
-		return nil, fmt.Errorf("Istio CA address is empty")
+		return nil, fmt.Errorf("istio CA address is empty")
```
Comment (myidpt) [inline security/cmd/node_agent/na/nodeagent.go]: Istio is used with the first letter capitalized. Can we ignore this?

Comment (JimmyCYJ) [reply security/cmd/node_agent/na/nodeagent.go]: Sure. Thanks for letting me know.

**Pull request merged**